packs/feats/skilled-climber.json

```diff
@@ -31,8 +31,16 @@
         "rules": [
             {
                 "key": "BaseSpeed",
-                "type": "climb",
+                "selector": "climb",
                 "value": 10
+            },
+            {
+                "key": "BaseSpeed",
+                "predicate": [
+                    "feat:scuttle-up"
+                ],
+                "selector": "climb",
+                "value": "@actor.attributes.speed.total"
```

Comment:
This will need a toggle since it only applies when "climbing trees or cavern walls". I recommend you use a toggleable roll option predicated on `feat:scuttle-up` and then predicate this BaseSpeed on just that toggle.

Reply:
Just committed a change for that